lib/create-organization-api.js

```diff
@@ -371,9 +374,48 @@ export default function createOrganizationApi ({ http }) {
       .then((response) => wrapAppDefinition(http, response.data), errorHandler)
   }
 
+  /**
+   * Gets a User
+   * @memberof ContentfulOrganizationAPI
+   * @param {string} id - User ID
+   * @return {Promise<User.User>} Promise for a User
+   * @example
+   * const contentful = require('contentful-management')
+   *
+   * client.getOrganization('<organization_id>')
+   * .then((organization) => organization.getUser('id'))
+   * .then((user) => console.log(user))
+   * .catch(console.error)
+   */
+  function getUser (id) {
+    return http
+      .get('users/' + id, {headers})
```

Comment:
@vbeleuta out of curiosity - what headers are here?

Reply:
there is a const headers defined at the beginning of the file:
const headers = {'x-contentful-enable-alpha-feature': 'organization-user-management-api'}